app/bundles/CoreBundle/Assets/css/libraries/libraries.less

```diff
@@ -3,6 +3,9 @@
 @import "bootstrap/bootstrap-vertical-tabs";
 @import "bootstrap/bootstrap-stacked-modals";
 
+//Remix
+@import "../../../../../../node_modules/remixicon/fonts/remixicon.css";
```

Comment:
Untested, but I think this could be changed to just `remixicon/fonts/remixicon`?